pkg/runner/runner.go

```diff
@@ -124,3 +132,17 @@ func (r *Runner) getVersions(release, channel, arch string) (siVersion, cloudVer
 	}
 	return
 }
+
+func (r *Runner) cleanupActions(options *flags.Options) (err error) {
```

Comment:
here also I don't understand why not just cleanup instead of cleanupActions. Or maybe removeImages.